### DISCUSSION TITLE
how can I get IoU score from my output

### DISCUSSION THREAD

**Discussion by GoldFiiish** [ultralytics/yolov5#4355] [Q&A]
I have used this code on my medical dataset and I wanted to get IoU score from it. mAP score is provided in run/train/exp/results.png but I couldn't find the related part for the IoU score.

**Comment by glenn-jocher** [comment]
@GoldFiiish this metric you refer to does not exist.

**Comment by glenn-jocher** [comment]
@GoldFiiish YOLOv5 uses established metrics of Precision, Recall, mAP@0.5 and mAP@0.5:0.95. If you'd like to add additional metrics please submit a PR updating metrics.py.

The fastest and easiest way to incorporate your ideas into the official codebase is to submit a Pull Request (PR) implementing your idea, and if applicable providing before and after profiling/inference/training results to help us understand the improvement your feature provides. This allows us to directly see the changes in the code and to understand how they affect workflows and performance.

Please see our ✅ [Contributing Guide](https://github.com/ultralytics/yolov5/blob/master/CONTRIBUTING.md) to get started.
